Lib/test/test_codecs.py

```diff
@@ -3401,5 +3402,19 @@ def test_rot13_func(self):
             'To be, or not to be, that is the question')
 
 
+class ZNormalizedTest(unittest.TestCase):
```

Comment:
I phan use `class NoarmalizedTest`, but I found that some cases would be failed after `codecs.unregister()`.
so I use a prefix of `Z` to avoid failed cases. Ugly way~

Reply:
ok, without calling  `codecs_unregister()`, it's fine too.